core/src/toga/window.py

```diff
@@ -72,6 +72,14 @@ def __init__(
 
         self.on_close = on_close
 
+    @property
+    def screen(self):
+        return self._impl.get_current_screen()
```

Comment:
This means the impl layer is returning interface objects. Dereferencing from impl to interface should be done in the interface layer.

Reply:
Done